src/claim/definitions.js

```diff
@@ -0,0 +1,455 @@
+const { definitions: ucaDefinitions } = require('@identity.com/uca');
+
+
+// ######################################### DEFINITIONS ###########################################
+const definitions = [
+  {
+    identifier: 'claim-cvc:Email.domain-v1',
+    version: '1',
+    type: 'cvc:Type:domain',
+    credentialItem: true,
+  },
+  {
+    identifier: 'claim-cvc:Email.properties-v1',
```

Comment:
Change this to claim-cvc:Email-v1

Reply:
See phoneNumber

Reply:
This was done to make the claim path work as composite UCA, so changing this may affect the claim path.

Reply:
I guess @dmelosantos is right. We should not change...

Reply:
Solved